Example/ViewController.swift

```diff
@@ -99,12 +99,12 @@ final class ViewController: UIViewController {
         ]
     }
 
-    private var state: PlayerState? {
+    private var state: ModernAVPlayerState? {
```

Comment:
As a user, I prefer to have to play with PlayerState instead of ModerAVPlayerState. 
In fact, I would prefer to inverse PlayerState (public enum) & ModernAVPlayerState (private pod class)